bears/documentation/DocumentationStyleBear.py

```diff
@@ -92,7 +96,16 @@ def process_documentation(self,
             new_desc = new_desc.rstrip() + '\n'
 
             new_metadata.append(m._replace(desc=new_desc.lstrip(' ')))
-        return (new_metadata, warning_desc)
+        new_comment = DocumentationComment.from_metadata(
+            new_metadata, doc_comment.docstyle_definition,
+            doc_comment.marker, doc_comment.indent, doc_comment.position)
+
+        if file[doc_comment.range.start.line - 2] != '\n':
```

Comment:
Quick qustion: I haven't been following your work, like at all, but a comment could potentially start in the 0th or 1st line, right? Then you'd be checking the last/last-but-one line (0 - 2 = -2), which is probably not what you intended over here (same applies to the if-clause below).